.travis.yml

```diff
@@ -26,6 +26,7 @@ before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runflake8
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./runisort; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./runcheckmigration; fi
```

Comment:
can you try removing the if and let the check happen on both python 2 and python 3? The previous line is a hack to avoid wasting too much time on differences between isort on python2 and 3 which have happened from time to time, but with the script you wrote this shouldn't happen right?